lab-carolina/karma.conf.js

```diff
@@ -0,0 +1,75 @@
+// Karma configuration
+// Generated on Mon Feb 06 2017 18:22:13 GMT-0800 (PST)
+const webpackConfig = require('./webpack.config.js');
+// webpackConfig.entry = {}; //no longer need with webpack upgrade
```

Comment:
You do want to delete the entry obj from the webpack.config. We're not testing that here in these unit tests.